CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedIO.java

```diff
@@ -303,18 +317,32 @@ private static void loadParameterFile(final DelimitedImportController importCont
                     }
 
                     definitions.add(impdef);
+
+                    if (!missingUserAttributes.isEmpty()) {
+                        final String message = String.format("A possible Template Error occured. In the tab `Run %d` following attributes are considered user added "
+                                + "for the destination `%s` with %s, as specified in the template. Hence they require `attribute_type` property. \n\n %s \n\n Consider using a new "
+                                + "template or add the missing attributes manually.", tabCount, schemaFactory.getLabel(),
+                                (showAllSchemaAttributes ? "`showAllSchemaAttributes` enabled" : "`showAllSchemaAttributes` disabled"), Arrays.toString(missingUserAttributes.toArray()));
+
+                        NotifyDisplayer.displayAlert(LOAD_TEMPLATE, "Template Error", message, Alert.AlertType.WARNING);
+                    }
                 }
 
                 importController.setClearManuallyAdded(false);
                 try {
-                    ((DelimitedImportPane) importController.getStage()).update(importController, definitions);
+                    ((DelimitedImportPane) importController.getImportPane()).update(importController, definitions);
                 } finally {
                     importController.setClearManuallyAdded(true);
                 }
             } else {
                 final String message = String.format("Can't find schema factory '%s'", destination);
                 NotifyDisplayer.displayAlert(LOAD_TEMPLATE, "Destination Schema Error", message, Alert.AlertType.ERROR);
             }
+        } catch (final NullPointerException ex) {
```

Comment:
I think my preference would be to do null checks over catching a null pointer as it's better practice that way. Sonar will also be flagging this. Potentially what you could is do a null check and then throw a different kind of exception (probably runtime) in the else block

Reply:
I tried to avoid checking null in ~17 lines which are unlikely/less likely to occur. NullPointerException is thrown at the JsonNode.get() hence cannot throw another exception and handle it in this code.